doc/code-generator/build.gradle

```diff
@@ -0,0 +1,138 @@
+import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
+
+plugins {
+  id 'java'
+  id 'org.openapi.generator'
+  id 'maven-publish'
+}
+
+group = "com.wci.termhub"
+version = "1.0.0-SNAPSHOT"
+
+// Input Specification locations. When using the inputFile, make sure you've updated
+// the doc with the changes you want to generate
+def inputURL = 'https://api.terminologyhub.com/terminology/doc/v3/api-docs'
+def inputFile = '$projectDir/src/th-swagger.yaml'
+
+repositories {
+  mavenCentral()
+}
+
+sourceCompatibility = 17
+targetCompatibility = 17
+
+test {
+  useJUnitPlatform()
+}
+
```

Comment:
Remove commented out code

Reply:
The commented our code is a placholder for the next languages we are going to implement. Since we know they will be supported and I was able to investigate all of them at once I wrote the code to just comment back in and generate the code immediately.